kv.go

```diff
@@ -0,0 +1,142 @@
+package geche
+
+import (
+	"sync"
+)
+
+type trieNode struct {
+	c        byte
+	next     [256]*trieNode
```

Comment:
Let's try a map here. Looks like it will be the same time complexity, but much better space complexity

Reply:
With "blind" iteration over 256 keys in dfs?

Reply:
Yep! Ehrlich gesagt, you can try to make it by linked list + map, or yes, try to find key letter by iterating over unordered map's keys. I'll be interested to see benchmarks :D